config/config/sc-config.yaml

```
@@ -100,6 +100,9 @@ falco:
       - macro: user_known_contact_k8s_api_server_activities
         append: true
         condition: or (container.image.repository in (ghcr.io/dexidp/dex))
+    opensearch.yaml: |-
+      - macro: user_privileged_containers
+        condition: container.image.repository=docker.io/elastisys/curl-jq and k8s.pod.name startswith opensearch-master-
```

```suggestion
        condition: container.image.repository=docker.io/elastisys/curl-jq and k8s.pod.name startswith opensearch-
```
This might be better in case OS runs with separate data and client pods, else it looks good.